examples/ex03_proc/test/ProcFL_test.py

```diff
@@ -156,11 +156,11 @@ def test_srl_rand_delays( s, dump_vcd ):
     asm_test( inst_bne.gen_value_test             ),
     asm_test( inst_bne.gen_random_test            ),
   ])
-  def test_bne( s, name, test, dump_vcd ):
+  def test_bne( s, name, test, cmdline_opts ):
```

Comment:
why do you need cmdline_opts in FL tests?

Reply:
because `dump_vcd` is now part of the unified `cmdline_opts`? also i think we initially had this in FL because we want to reuse tests across multiple levels?

Reply:
hmmm right, we don't really need dump_vcd for FL model but yeah, reusing tests...